lib/infra_conversion_throttler.rb

```diff
@@ -10,14 +10,21 @@ def self.start_conversions
       _log.debug("-- Currently running jobs in EMS: #{running}")
       slots = (ems.miq_custom_get('MaxTransformationRunners') || Settings.transformation.limits.max_concurrent_tasks_per_ems).to_i - running
 
-      if slots <= 0
-        _log.debug("-- No available slot in EMS. Stopping.")
-        next
-      end
-      _log.debug("-- Available slots in EMS: #{slots}")
-
       jobs.each do |job|
         vm_name = job.migration_task.source.name
+
+        preflight_check = job.migration_task.preflight_check
+        if preflight_check[:status] == 'Error'
+          _log.info("Preflight check for #{vm_name} has failed. Discarding.")
```

Comment:
Minor nit, I'd change that to a `_log.warn` or `_log.error`.

Reply:
Changed to `_log.error`.